<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>652</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>FTP сервер</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/res/app.ico</normaloff>:/res/app.ico</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QTabWidget" name="tabWidget">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab">
         <attribute name="title">
          <string>Лог</string>
         </attribute>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QTextEdit" name="textlog">
            <property name="readOnly">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_2">
            <item>
             <widget class="QPushButton" name="start">
              <property name="text">
               <string>Запуск сервера</string>
              </property>
              <property name="icon">
               <iconset resource="res.qrc">
                <normaloff>:/res/start.png</normaloff>:/res/start.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="stop">
              <property name="text">
               <string>Остановить сервер</string>
              </property>
              <property name="icon">
               <iconset resource="res.qrc">
                <normaloff>:/res/stop.png</normaloff>:/res/stop.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="clearLogBtn">
              <property name="text">
               <string>Очистить лог</string>
              </property>
              <property name="icon">
               <iconset resource="res.qrc">
                <normaloff>:/res/clear.png</normaloff>:/res/clear.png</iconset>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="maximumSize">
                 <size>
                  <width>1000</width>
                  <height>100</height>
                 </size>
                </property>
                <property name="text">
                 <string>Число клиентов</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLCDNumber" name="clientslcd">
                <property name="maximumSize">
                 <size>
                  <width>64</width>
                  <height>64</height>
                 </size>
                </property>
                <property name="layoutDirection">
                 <enum>Qt::RightToLeft</enum>
                </property>
                <property name="autoFillBackground">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_2">
         <attribute name="title">
          <string>Настройки</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="Utf8Cb">
            <property name="text">
             <string>Принудительный UTF-8</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Пользователи</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QTableWidget" name="usersTable">
            <property name="editTriggers">
             <set>QAbstractItemView::NoEditTriggers</set>
            </property>
            <property name="showGrid">
             <bool>false</bool>
            </property>
            <column>
             <property name="text">
              <string>Имя</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Пароль</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Папка</string>
             </property>
            </column>
            <column>
             <property name="text">
              <string>Аттрибуты</string>
             </property>
            </column>
           </widget>
          </item>
          <item>
           <widget class="QSplitter" name="splitter">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <widget class="QToolButton" name="addUserButton">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/res/user_add.png</normaloff>:/res/user_add.png</iconset>
             </property>
            </widget>
            <widget class="QToolButton" name="editUserButton">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/res/user_edit.png</normaloff>:/res/user_edit.png</iconset>
             </property>
            </widget>
            <widget class="QToolButton" name="deleteUserButton">
             <property name="text">
              <string>...</string>
             </property>
             <property name="icon">
              <iconset resource="res.qrc">
               <normaloff>:/res/user_delete.png</normaloff>:/res/user_delete.png</iconset>
             </property>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>64</width>
          <height>64</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::RightToLeft</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>5</number>
        </property>
        <property name="midLineWidth">
         <number>0</number>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="pixmap">
         <pixmap resource="res.qrc">:/res/red.png</pixmap>
        </property>
        <property name="scaledContents">
         <bool>true</bool>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTop|Qt::AlignTrailing</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>652</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Сервер</string>
    </property>
    <addaction name="actionStartServer"/>
    <addaction name="actionStopServer"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Помощь</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionStartServer">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/start.png</normaloff>:/res/start.png</iconset>
   </property>
   <property name="text">
    <string>Запуск</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionStopServer">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/stop.png</normaloff>:/res/stop.png</iconset>
   </property>
   <property name="text">
    <string>Остановка</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/res/about.png</normaloff>:/res/about.png</iconset>
   </property>
   <property name="text">
    <string>О программе</string>
   </property>
   <property name="shortcut">
    <string>F1</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Закрыть программу</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
